v0/destinations/pardot/transform.js

```diff
@@ -0,0 +1,56 @@
+const { EventType } = require("../../../constants");
+const { CONFIG_CATEGORIES, MAPPING_CONFIG } = require("./config");
+const {
+  defaultRequestConfig,
+  getFieldValueFromMessage,
+  constructPayload,
+  defaultPostRequestConfig
+} = require("../../util");
+
+const responseBuilderSimple = (message, category, destination) => {
+  const payload = {};
+  const contact = constructPayload(message, MAPPING_CONFIG[category.name]);
+  payload.contact = contact;
+  if (payload) {
+    const responseBody = { ...payload, apiKey: destination.Config.apiKey };
+    const response = defaultRequestConfig();
+    response.endpoint = String(destination.Config.apiUrl).concat(
+      String(category.endPoint)
+    );
+    response.method = defaultPostRequestConfig.requestMethod;
+    response.headers = {
+      "Content-Type": "application/json",
+      "Api-Token": destination.Config.apiKey
+    };
+    response.userId = getFieldValueFromMessage(message, "userId");
+    response.body.JSON = responseBody;
+    return response;
+  }
+  // fail-safety for developer error
+  throw new Error("Payload could not be constructed");
+};
+
+// Consider only Identify call for Pardot for now
+
+const processEvent = (message, destination) => {
+  if (!message.type) {
+    throw Error("Message Type is not present. Aborting message.");
+  }
+  const messageType = message.type.toLowerCase();
+
+  let category;
+  switch (messageType) {
+    case EventType.IDENTIFY:
+      category = CONFIG_CATEGORIES.IDENTIFY;
```

Comment:
We're having only one possible value of the variable `category`. It's better to use it as a constant. 

We can directly export the identify config from `config.js` and use it directly in the `responseBuilder` method in stead of passing the mapping JSON explicitly